third_party/ienn/src/ie_compilation.cpp

```diff
@@ -289,7 +287,17 @@ int32_t Compilation::AddElementwise(const Operation& operation) {
       return error_t::BAD_DATA;
     }
     const uint32_t output_index = operation.outputs[0];
-    index_op_map_[output_index] = elementwise_node->output(0);
+    if (params.fuse_code != fuse_t::FUSED_NONE) {
+      std::shared_ptr<ngraph::Node> activation_node;
+      result = AddActivationByFusedCode(params.fuse_code, elementwise_node,
+                                        activation_node);
+      if (result != error_t::NOT_ERROR) {
+        return result;
+      }
+      index_op_map_[output_index] = activation_node->output(0);
+    } else {
```

Comment:
I think `else` here may be not necessary. Keep line 192 and then line 297 will reset the `index_op_map_[output_index]` and will make codes more brief. Just personal opinion~

Reply:
I agree with you.